indigo/views.py

```diff
@@ -430,6 +439,50 @@ class OrganisationListDownload(ModelListDownload):
 class PipelineListDownload(ModelListDownload):
     _model = Pipeline
 
+    def _get_extra_headers(self):
+        return ["Delivery Locations - Name", "Delivery Locations - Country"]
+
+    def _get_extra_data(self, data):
+        delivery_locations_list = jsonpointer.resolve_pointer(
+            data.data_public, "/delivery_locations", []
+        )
+        if isinstance(delivery_locations_list, list):
+            delivery_location_names = [
+                jsonpointer.resolve_pointer(d, "/location_name/value", "")
+                for d in delivery_locations_list
+            ]
+            delivery_location_country_codes = [
+                jsonpointer.resolve_pointer(d, "/location_country/value", "")
+                for d in delivery_locations_list
+            ]
+            # List/set removes duplicates
```

Comment:
Yes. While I haven't asked specifically about that I did check with them about things being across 2 columns oddly. 

If people download a ZIP file they will get all the raw data anyway, so I think it will be ok.